charts/cluster-scanner/templates/deployment.yaml

```diff
@@ -53,6 +53,40 @@ spec:
           configMap:
             name: {{ include "cluster-scanner.fullname" . }}-registry-mirrors
         {{- end }}
+        {{- if ( include "cluster-scanner.nats.tls.hasSelfSignedCert" . ) }}
+        - name: js-certs
+          secret:
+            secretName: {{ include "cluster-scanner.fullname" . }}
+            items:
+              - key: js_tls_key
+                path: js_tls.key
+              - key: js_tls_cert
+                path: js_tls.cert
+              - key: js_tls_ca
+                path: js_tls.ca
+        {{- end }}
+        {{- if ( include "cluster-scanner.nats.tls.hasCustomCert" . ) }}
```

Comment:
An `{{- else if ( include "cluster-scanner.nats.tls.hasCustomCert" . ) }}` would help the readability to me

Reply:
Done: https://github.com/sysdiglabs/charts/pull/1630/commits/2ab13fdf633658c58ed73ba2a6b1997f8a2f93ec